static/app/routes.tsx

```diff
@@ -2037,7 +2045,8 @@ function buildRoutes() {
           component={hoc(
             make(
               () =>
-                import('sentry/views/issueDetails/groupEventDetails/groupEventDetails')
+                import('sentry/views/issueDetails/groupEventDetails/groupEventDetails'),
+              <GroupEventDetailsLoading />
```

Comment:
without this it displays the spinner while waiting for react to render this component for the first time (big slow component)

Reply:
Is this basically just overriding the spinner?

Reply:
yeah